packages/server/src/integration-test/postgres.spec.ts

```diff
@@ -151,30 +197,105 @@ describe("row api - postgres", () => {
     value: number
   }
 
+  type ForeignTableInfo = {
+    table: Table
+    fieldName: string
+    relationshipType: RelationshipTypes
```

Comment:
Property is singular but the type is plural - which one is it?

Reply:
This enumeration needs refactored - enums should all be singular (just an old one that wasn't that way). `RelationshipTypes` should really just be `RelationshipType`.

Reply:
I will not change this here, as this enum is used in a lot of places, svelte included